bin/common.sh

```diff
@@ -31,44 +32,47 @@ function output_software_info {
 #   GIT_PROVIDER - platform for git hosting
 #   GIT_REPO_NAME - repo in "user/repository" format
 #######################################
-function detect_git_provider
-{
-# Check if it is GitHub
-# https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
-if [ ! -z ${GITHUB_REPOSITORY+h} ];
-        then DEBUG "GITHUB_REPOSITORY variable is set to $GITHUB_REPOSITORY"
-            INFO "Assuming we are running GitHub Action";
-            readonly GIT_PROVIDER="github"
-            readonly GIT_REPO_NAME=$GITHUB_REPOSITORY
-    else
-        DEBUG "GITHUB_REPOSITORY variable is NOT set assuming we are NOT running GitHub Action";
-fi
+function detect_git_provider {
```

Comment:
Please, fix wrong indentation in this function

Reply:
Done!